src/views/mfa-verify/PushForm.js

```diff
@@ -62,7 +79,9 @@ define(['okta'], function (Okta) {
       }
     },
     submit: function (e) {
-      e.preventDefault();
```

Comment:
We probably don't want to skip the `setSubmitState(false)` portion in `submit()` b/c we also want to update push button label?